corehq/apps/hqwebapp/templates/hqwebapp/includes/bootstrap5/global_navigation_bar.html

```diff
@@ -64,7 +64,7 @@
           {% endif %}
           {% if user.is_superuser %}
           <li>
-            <a href="{% url "feature_flags_and_privileges" domain %}" class="track-usage-link"
+            <a href="{% url "feature_flags_and_privileges" domain %}" class="dropdown-item track-usage-link"
                data-category="Nav Bar" data-action="Gear Icon" data-label="Current Subscription">
               <i class="fa fa-flag dropdown-icon"></i> {% trans "Feature Flags and Privileges" %}
```

Comment:
@biyeun I see the other icons have two classes, eg `fa-dashboard` and `icon-dashboard`.  Should there also be a `icon-flag` class here?  What is the point of those dual classes?